Makefile

```diff
@@ -3,38 +3,16 @@
 clean:
 	kne delete kne/twodut_topo.pb.txt; kne delete kne/topo.pb.txt; kne delete kne/twodut_oneotg_topo.pb.txt; exit 0
 
```

Comment:
Don't need this anymore?

Reply:
might be still be useful, changed to find so no need to add new paths manually